kubeflow/trainer/api/trainer_client.py

```diff
@@ -12,526 +12,137 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import copy
 import logging
-import multiprocessing
-import queue
-import random
-import string
-import time
-import uuid
-from typing import Dict, List, Optional, Union, Set
+from typing import Dict, Optional, Union, Set, List
 
 from kubeflow.trainer.constants import constants
 from kubeflow.trainer.types import types
-from kubeflow.trainer.utils import utils
-from kubeflow_trainer_api import models
-from kubernetes import client, config, watch
+from kubeflow.trainer.types.backends import K8SBackendConfig, LocalProcessBackendConfig
```

Comment:
I suggest we keep the full name, not all users might be aware of k8s acronym.
```suggestion
from kubeflow.trainer.types.backends import KubernetesBackendConfig, LocalProcessBackendConfig
```

Reply:
done